src/sentry_plugins/slack/plugin.py

```diff
@@ -189,7 +189,7 @@ def notify(self, notification):
                 fields.append(
                     {
                         'title': 'Triggered By',
-                        'value': ', '.join('<%s | %s>' % r for r in rules),
+                        'value': u', '.join(u'<{} | {}>'.format(*r) for r in rules),
```

Comment:
`r` is a two item tuple setup here

https://github.com/getsentry/sentry-plugins/blob/7a6824b75860faebc461c20db6c90b40ff0111e0/src/sentry_plugins/slack/plugin.py#L186